### DISCUSSION TITLE
Steamtools打开解锁模式后乱码无法解锁

### DISCUSSION THREAD

**Discussion by tiantang111** [pjy612/SteamManifestCache#864] [Question / Answer]
Steamtools打开解锁模式后乱码无法解锁
尝试过重装tools以及重装steam，但多次卸载重装无用，多处询问无果，只能再次询问
![Desktop Screenshot 2024 10 07 - 19 25 37 34](https://github.com/user-attachments/assets/490db5df-4b09-43c6-804f-a1b0935f6017)

**Comment by pjy612** [comment]
tools的问题 问tools 去。
又不是我写的 tools
